src/scripts/Providers/OSUI/SharedProviderResources/Flatpickr/scss/_flatpickr.scss

```diff
@@ -658,3 +659,133 @@ body:has(.has-accessible-features) .osui-monthpicker__dropdown {
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.flatpickr-rContainer,
+	.flatpickr-days {
+		padding: 5px;
+	}
+
+	.flatpickr-current-month {
+		.flatpickr-monthDropdown-months {
+			appearance: revert;
+
+			&:focus,
+			&:active {
+				outline: 3px solid var(--color-focus-outer);
+			}
+		}
+	}
+
+	.flatpickr-months {
+		.flatpickr-prev-month,
+		.flatpickr-next-month {
+			&:focus,
+			&:active {
+				outline: 3px solid var(--color-focus-outer);
+			}
+		}
+	}
+
+	.flatpickr-day {
+		border: none;
+		position: relative;
+		z-index: 0;
+
+		&:hover,
+		&:focus,
+		&:focus-visible {
+			z-index: 2;
```

Comment:
Please review what would be the correct layer to use here

Reply:
Done!